src/amo/sagas/reviews.js

```diff
@@ -51,6 +54,35 @@ function* fetchReviews({
   }
 }
 
+function* fetchUserReviews({
+  payload: { errorHandlerId, page, userId },
+}: FetchUserReviewsAction): Generator<any, any, any> {
+  const errorHandler = createErrorHandler(errorHandlerId);
+
+  try {
+    const state = yield select(getState);
+
+    const response = yield call(getReviews, {
```

Comment:
Huh. It looks like this saga never got Flow types. That's a bummer because when I added them, I see some subtle bugs in all of these API calls 👻 :

```diff
diff --git a/src/amo/api/reviews.js b/src/amo/api/reviews.js
index e46fb8e97..003d6a5c2 100644
--- a/src/amo/api/reviews.js
+++ b/src/amo/api/reviews.js
@@ -135,7 +135,7 @@ export const replyToReview = ({
   });
 };
 
-type GetReviewsParams = {|
+export type GetReviewsParams = {|
   // This is the addon ID, slug, or guid.
   addon?: number | string,
   apiState?: ApiStateType,
diff --git a/src/amo/sagas/reviews.js b/src/amo/sagas/reviews.js
index 1094e2281..7a7ebe55c 100644
--- a/src/amo/sagas/reviews.js
+++ b/src/amo/sagas/reviews.js
@@ -21,6 +21,7 @@ import {
 } from 'amo/constants';
 import log from 'core/logger';
 import { createErrorHandler, getState } from 'core/sagas/utils';
+import type { GetReviewsParams } from 'amo/api/reviews';
 import type {
   FetchReviewsAction,
   FetchUserReviewsAction,
@@ -62,13 +63,14 @@ function* fetchUserReviews({
   try {
     const state = yield select(getState);
 
-    const response = yield call(getReviews, {
+    const params: GetReviewsParams = {
       api: state.api,
       // Hide star-only ratings (reviews that do not have a body).
       filter: 'without_empty_body',
       page,
       user: userId,
-    });
+    };
+    const response = yield call(getReviews, params);
 
     yield put(
       setUserReviews({
```

Reply:
wow, thank you!